<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsDoc</class>
 <widget class="QMainWindow" name="GraphicsDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphics</string>
  </property>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Данные</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Открыть</string>
     </property>
     <addaction name="action_2"/>
     <addaction name="action_LoadOrion"/>
    </widget>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>График</string>
    </property>
    <addaction name="actionAddAxe"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarPanel">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/Resources/images/open.png</normaloff>:/Resources/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="graphics.qrc">
     <normaloff>:/Resources/images/save.png</normaloff>:/Resources/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>САПР</string>
   </property>
  </action>
  <action name="action_LoadOrion">
   <property name="text">
    <string>Орион</string>
   </property>
  </action>
  <action name="actionAddAxe">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
